cmd/comparator-rest/docs/openapi.yaml

```diff
@@ -69,43 +69,20 @@ paths:
     post:
       tags:
         - required
+      description: Evaluates an operator with its inputs and returns the result.
       requestBody:
         required: true
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/Comparison"
-            example: {
```

Comment:
Please don't delete the examples - they're there to illustrate usage of the API.

Reply:
done